internal/plugin/command_install.go

```diff
@@ -103,16 +103,16 @@ func installPlugin(manifest *Manifest, repositoryDir, installDir string) error {
 	var pluginInstaller plugin.PluginInstaller
 	switch language {
 	case "go":
-		pluginInstaller = &plugin.GoPluginInstaller{Name: manifest.Name}
+		pluginInstaller = &plugin.GoPluginInstaller{Name: manifest.Id}
```

Comment:
Does this need to be changed to "Id" now?
```suggestion
		pluginInstaller = &plugin.GoPluginInstaller{Id: manifest.Id}
```